Source/Examples/ExampleLibrary/Issues/Issues.cs

```diff
@@ -2283,6 +2283,38 @@ public static PlotModel EmojiText()
 
             return plot;
         }
+        [Example("#1512: FindWindowStartIndex.")]
+        public static PlotModel FindWindowsStartIndex()
+        {
+            var plotModel1 = new PlotModel { Title = "AreaSeries broken in time" };
+            var axis = new LinearAxis {Position = AxisPosition.Left, MinimumPadding = 0, MaximumPadding = 0.06, AbsoluteMinimum = 0};
+            var xAxis = new LinearAxis() {Position = AxisPosition.Bottom};
```

Comment:
Could this have `Minimum = 4` set, so that the error is presented immediately? (Or otherwise, a subtitle explaining how to provoke the issue). In the future it won't necessarily be easy for people to find this PR and read the discussion relevant to the example.